arrocke/gloss-translation
# feat: finish gloss import with lambda

## What has changed

* new database record for tracking import jobs
* import lambda and api request to start import are connected by an SQS queue.
* frontend page shows different UI based on whether the import is running, idle or complete.
* Everything is configured in AWS, and I've tested that the lambda code runs when something is put in the queue



## Connected Issues

closes #134 



## QA Steps

1. Comment out the SQS queue code in the post import request. This will allow you to test the lambda locally without SQS.
2. Import the spanish glosses to a language
3. Run `nx serve lambda-functions`
4. When the lambda completes, verify that the import view shows success screen



## Post-Deployment



- Upload the latest lambda code to AWS
